include/odp/api/spec/queue_types.h

```diff
@@ -130,14 +131,16 @@ typedef struct odp_queue_capability_t {
 	  * types are used simultaneously. */
 	uint32_t max_queues;
 
-	/** Maximum number of ordered locks per queue */
-	uint32_t max_ordered_locks;
+	/** @deprecated Use max_ordered_locks field of
+	 * odp_schedule_capability_t instead */
+	uint32_t ODP_DEPRECATE(max_ordered_locks);
```

Comment:
Checkpatch complains about this:
```
---------------------------------------------------------------
0001-api-queue-schedule-move-scheduler-capabilities-to-sc.patch
---------------------------------------------------------------
WARNING: function definition argument 'max_ordered_locks' should also have an identifier name
#37: FILE: include/odp/api/spec/queue_types.h:136:
+	uint32_t ODP_DEPRECATE(max_ordered_locks);
WARNING: Prefer 'unsigned int' to bare use of 'unsigned'
#43: FILE: include/odp/api/spec/queue_types.h:140:
+	unsigned ODP_DEPRECATE(max_sched_groups);
WARNING: function definition argument 'max_sched_groups' should also have an identifier name
#43: FILE: include/odp/api/spec/queue_types.h:140:
+	unsigned ODP_DEPRECATE(max_sched_groups);
WARNING: Prefer 'unsigned int' to bare use of 'unsigned'
#48: FILE: include/odp/api/spec/queue_types.h:143:
+	unsigned ODP_DEPRECATE(sched_prios);
WARNING: function definition argument 'sched_prios' should also have an identifier name
#48: FILE: include/odp/api/spec/queue_types.h:143:
+	unsigned ODP_DEPRECATE(sched_prios);
total: 0 errors, 5 warnings, 0 checks, 68 lines checked
NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.
0001-api-queue-schedule-move-scheduler-capabilities-to-sc.patch has style problems, please review.
```

Reply:
@Bill-Fischofer-Linaro checkpatch does not understand `ODP_DEPRECATE()` syntax, thus it thinks that ODP_DEPRECATE is a function with K&R style arguments. Also I don't think we should change types of old fields.

Reply:
"unsigned" is a synonym of "unsigned int", so we can update the type without any one noticing it.

Reply:
fixed `unsigned` -> `unsigned int`.